packages/appsync-emulator-serverless/lambdaRunnerGo.js

```diff
@@ -0,0 +1,66 @@
+const {
+  log,
+  sendErr,
+  sendOutput,
+  installExceptionHandlers,
+} = require('./lambda/util');
+
+const path = require('path');
+
+process.once(
+  'message',
+  async ({ serverlessDirectory, handlerMethod, payload }) => {
+    try {
+      log.info('load', handlerMethod);
+
+      const { spawn } = require('child_process');
+      const args = ['local', 'invoke'];
+
+      const sam = spawn('sam', args, {
+        env: process.env,
+        shell: '/bin/bash',
+        cwd: path.join(serverlessDirectory, handlerMethod),
+      });
+
+      sam.stdin.write(`${JSON.stringify(payload)}\n`);
```

Comment:
Everything from here below looks similar to the python lambda handler, should we extract out common functionality to a single place ?